libjulia/backends/evm/EVMCodeTransform.cpp

```diff
@@ -545,11 +609,20 @@ void CodeTransform::expectDeposit(int _deposit, int _oldHeight) const
 void CodeTransform::checkStackHeight(void const* _astElement) const
 {
 	solAssert(m_info.stackHeightInfo.count(_astElement), "Stack height for AST element not found.");
-	solAssert(
-		m_info.stackHeightInfo.at(_astElement) == m_assembly.stackHeight() - m_stackAdjustment,
-		"Stack height mismatch between analysis and code generation phase: Analysis: " +
-		to_string(m_info.stackHeightInfo.at(_astElement)) +
-		" code gen: " +
-		to_string(m_assembly.stackHeight() - m_stackAdjustment)
-	);
+	if (m_language == AssemblyStack::Language::StrictAssembly)
+		solAssert(
+			m_info.stackHeightInfo.at(_astElement) >= m_assembly.stackHeight() - m_stackAdjustment,
```

Comment:
Perhaps it would better to just ignore it?

Reply:
It is still useful somehow to ensure that reuse stack cannot use more stack than before, i.e. the stack height in codegen should be lower than the one from analysis.